src/components/Dropzone/Dropzone.vue

```diff
@@ -127,15 +132,28 @@ export default {
         this.on('removedfile', (file) => {
           vm.fileCnt = this.files.length;
         });
+
+        // Wait for uploading to complete, then submit the request
+        this.on('queuecomplete', () => {
+          // Check whether it's the failure case
+          if (!(this.files[0].status != Dropzone.SUCCESS && this.getQueuedFiles().length <= 1)) {
```

Comment:
Why the queued files should be no more than 1?

Reply:
If there is a file upload failure, this `queuecomplete` event will also be triggered. This is to check that we only submit in the case of successful submissions of two files.